roles/ash_cli/defaults/main.yml

```diff
@@ -2,7 +2,13 @@
 # Copyright (c) 2022-2023, E36 Knots
 ---
 # Ash CLI version
-ash_cli_version: 0.1.0
+ash_cli_version: 0.1.1
```

Comment:
I think we need to release this before merging this PR:

```
fatal: [validator03]: FAILED! => changed=false
  dest: /opt/avalanche/ash-cli/releases/ash-linux-amd64-v0.1.1
  elapsed: 0
  msg: Request failed
  response: 'HTTP Error 404: Not Found'
  status_code: 404
  url: https://github.com/AshAvalanche/ash-rs/releases/download/v0.1.1/ash-linux-amd64-v0.1.1.sha512
```

Reply:
Yep this is why I marked the other PR as dependency

Reply:
It should be good now